### DISCUSSION TITLE
Disable "write" highlights

### DISCUSSION THREAD

**Discussion by shuvashish76** [koreader/koreader#11558] [Q&A] [answered]
Each time I add/remove highlight/note it asks "Write highlights into this PDF?"
I don't wanna **modify** my PDF files, just normally save highlights in their respective `.sdr` folder as `.lua` file. How can I disable it so that it won't ask me for any PDF & similar files?

Edit: Also the reverse option, _write as always_ without asking for confirmation each time option would be nice for those who prefer that way.

**Comment by hius07** [comment, marked as answer]
Sure, you have both at Settings - Document - Save document (write highlights into PDF).

**Reply by shuvashish76** [reply]
Thanks.
Any other file formats that supports writing into the file?

**Reply by hius07** [reply]
With our Text editor plugin only.
